scriptmodules/helpers.sh

```diff
@@ -899,6 +902,21 @@ function loadModuleConfig() {
     done
 }
 
+## @fn getSystemExtensions()
+## @param system system name to fetch extensions for
+## @brief Prints a comma separated string of disk extensions supported by the system
+function getSystemExtensions() {
+    local platforms="$scriptdir/platforms.cfg"
+    local platforms_alt="$configdir/all/platforms.cfg"
+    [[ -f "$platform_alt" ]] && platforms="$platforms_alt"
+    
+    iniConfig "=" '"' "$platforms"
+    iniGet "$1_exts"
+
+    function join_by { local IFS="$1"; shift; echo "$*"; }
+    join_by , $ini_value
```

Comment:
a bash replace to replace the spaces with tabs would be simpler - but why not just return them as a string with space separated - can easily be converted to an array, to be processed later. (Also I can then use it for the addSystem function)

Also something like this would seem better

```
local config
for config in "$configdir/all/platforms.cfg" "$scriptdir/platforms.cfg"; do
     [[ -f "$config" ]] && break
done
```

would seem better.